src/main.cpp

```diff
@@ -1,6 +1,6 @@
 /*
 Makerfabs ESP32 UWB DW1000,
-Anchor code, for communication with single tag.
+Code for tag.
```

Comment:
It's actually code for both the Anchor and the Tag.

Reply:
True that